src/ci/docker/scripts/build-clang.sh

```diff
@@ -5,7 +5,7 @@ set -ex
 source shared.sh
 
 # Try to keep the LLVM version here in sync with src/ci/scripts/install-clang.sh
-LLVM=llvmorg-19.1.0-rc3
+LLVM=llvmorg-19.1.5
```

Comment:
I would move this update to a separate PR, just to avoid suprirses.

Reply:
Fixed, I'm not sure if @mrkajetanp had changed this deliberately in the `dist-aarch64/build-clang.sh`, so it might be worth confirming it all still works

Reply:
I bumped it when doing experiments with BOLT, but since we're not enabling bolt for the time being it's not needed as much. Though getting off the RCs if possible is probably good practice in general.